staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -589,6 +589,21 @@ type KubeletConfiguration struct {
 	// Default: 5
 	// +optional
 	ContainerLogMaxFiles *int32 `json:"containerLogMaxFiles,omitempty"`
+
+	// ContainerLogMaxWorkers specifies the maximum number of concurrent workers to spawn
+	// for performing the log rotate operations. Set this count to 0 for disabling the
+	// concurrent log rotation workflows
+	// Default: 0
+	// +optional
+	ContainerLogMaxWorkers *int32 `json:"containerLogMaxWorkers,omitempty"`
+
+	// ContainerLogMonitorInterval specifies the duration at which the container logs are monitored
+	// for performing the log rotate operation. This defaults to 10 * time.Seconds. But can be
+	// customized to a smaller value based on the log generation rate and the size required to be
+	// rotated against
+	// Default: 10s
+	// +optional
+	ContainerLogMonitorInterval *metav1.Duration `json:"containerLogMonitorInterval,omitempty"`
```

Comment:
Add validation to ValidateKubeletConfiguration that a positive number is specified here. Question for node folks, is there a minimum value that should be required, like 1s?

Reply:
I have added the validation. Will hold adding the min value check until I hear back on that from node folks.